modules/test/playwright/pages/commerce/commerce-channel-web/commerceAdminChannelDetailsPage.ts

```diff
@@ -327,6 +337,109 @@ export class CommerceAdminChannelDetailsPage {
 		await waitForAlert(await this.sidePanelNestedFrame('Shipping Methods'));
 	}
 
+	async addFixedTaxRate(amount: string, name: string) {
+		const tableName = 'Tax Calculations';
+
+		await (
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).click();
+
+		await expect(
+			(await this.sidePanelFrame(tableName)).getByLabel('Active')
+		).toBeVisible();
+
+		await (await this.sidePanelFrame(tableName))
+			.getByLabel('Active')
+			.check();
+		await (await this.frameSaveButton(false, tableName)).click();
+
+		await (await this.taxRatesTab(tableName)).click();
+		await (await this.sidePanelFrame(tableName))
+			.getByTitle('Add Tax Rate')
+			.click();
+
+		await expect(
+			(await this.addTaxRateFrame()).getByText('Tax Category')
+		).toBeVisible();
+
+		await (await this.addTaxRateFrame())
+			.getByLabel('Tax Category')
+			.selectOption(name);
+		await (await this.addTaxRateFrame()).getByLabel('Amount').fill(amount);
+		await (await this.addTaxRateFrame())
+			.getByRole('button', {name: 'Submit'})
+			.click();
+
+		await this.page.reload();
+
+		await expect(
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).toBeVisible();
+	}
+
+	async editFixedTaxRate(newAmount: string, name: string) {
+		const tableName = 'Tax Calculations';
+
+		await (
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).click();
+		await (await this.taxRatesTab(tableName)).click();
+
+		await (await this.getRowByTextFromSidePanelTable(tableName, name))
+			.locator('.dnd-td')
+			.nth(2)
+			.getByRole('button', {name: 'Actions'})
+			.click();
+
+		await (await this.sidePanelFrame(tableName))
+			.getByRole('menuitem', {name: 'Edit'})
+			.click();
+
+		await (await this.sidePanelNestedFrame(tableName))
+			.getByLabel('Amount')
+			.fill(newAmount);
+		await (await this.frameSaveButton(true, tableName)).click();
+
+		await this.page.reload();
```

Comment:
@smottal It's easier and less painful then closing everything (side panel, frame etc.), plus it makes sure that the page is updated with all the changes because it reloads it. Also if I remember correctly if I dont do this it can be a bit flaky but I'm not 100% sure bc this was some time ago.

Reply:
There is a check right away after it that's checking the changes:
await expect(
			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
		).toBeVisible();